test/parallel/test-btoa-atob.js

```diff
@@ -15,3 +15,16 @@ throws(() => buffer.btoa(), /TypeError/);
 
 strictEqual(atob(' '), '');
 strictEqual(atob('  YW\tJ\njZA=\r= '), 'abcd');
+
+strictEqual(atob(null), '\x9Eée');
+strictEqual(atob(NaN), '5£');
+strictEqual(atob(Infinity), '"wâ\x9E+r');
+strictEqual(atob(true), '¶»\x9E');
+strictEqual(atob(1234), '×mø');
+strictEqual(atob([]), '');
+strictEqual(atob({ toString: () => '' }), '');
+strictEqual(atob({ [Symbol.toPrimitive]: () => '' }), '');
+
+throws(() => atob(Symbol()), /TypeError/);
+[undefined, false, () => {}, 0, 1, 0n, 1n, -Infinity, [1], {}].forEach((value) =>
+  throws(() => atob(value), { constructor: DOMException }));
```

Comment:
Refs: https://github.com/nodejs/node/pull/42662#discussion_r846632275
```suggestion
  // See #2 - https://html.spec.whatwg.org/multipage/webappapis.html#dom-atob
  throws(() => atob(value), { constructor: DOMException, name: 'InvalidCharacterError' }));
```